fusor-ember-cli/app/routes/deployment.js

```diff
@@ -43,7 +43,7 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
             contentType: 'application/json',
             success: function(response) {
               resolve(response);
-              var uuid = response.id;
+              //var uuid = response.id;
```

Comment:
The response only returns an id when a request is sent to deploy an RHCI deployment, which doesn't happen in this workflow.